app/controllers/api/v1/entries_controller.rb

```diff
@@ -73,6 +74,17 @@ def upload_tsv
 
   private
 
+  def authenticate_token
+    raw_token = request.headers['Authorization']&.split(' ')&.last
+    token = AccessToken.find_by(token: raw_token)
+
+    if token && token.unexpired?
```

Comment:
liveやactiveというメソッド名はどうでしょうか？
また、safe opeartorが使えそうです。

```suggestion
    if token&.live?
```

や

```suggestion
    if token&.acitve?
```

の、イメージです。

Reply:
修正しました。
https://github.com/pubannotation/pubdictionaries/pull/142/commits/9a85cbafb68e5fc0354ec56449e2680edb34ba64